src/pages/workspaces/workspace/applications/AppLauncher.js

```diff
@@ -55,6 +62,85 @@ const ApplicationLauncher = _.flow(
   const runtime = getCurrentRuntime(runtimes)
   const runtimeStatus = getConvertedRuntimeStatus(runtime) // preserve null vs undefined
 
+  const FileOutdatedModal = ({ onDismiss, bucketName }) => {
+    const handleChoice = _.flow(
+      withErrorReportingInModal('Error setting up analysis file syncing')(onDismiss),
+      Utils.withBusyState(setBusy)
+    )(async shouldCopy => {
+      await Promise.all(_.flatMap(async analysis => {
+        const currentMetadata = analysis.metadata
+        const file = analysis.name.split('/')[1]
+        if (shouldCopy) {
+          currentMetadata[hashedOwnerEmail] = ''
+          await Ajax().Buckets.analysis(googleProject, bucketName, stripExtension(file), tools.RStudio.label).copyWithMetadata(stripExtension(file).concat(`_copy${Date.now().toString()}.`).concat(tools.RStudio.ext), bucketName, currentMetadata)
+        }
+        currentMetadata[hashedOwnerEmail] = 'doNotSync'
+        await Ajax().Buckets.analysis(googleProject, bucketName, stripExtension(file), tools.RStudio.label).updateMetadata(file, currentMetadata)
+      }, outdatedAnalyses))
+      onDismiss()
+    })
+
+    const getDisplayList = analyses => {
+      let res = ''
+      let i = 1
+      const end = analyses?.length
+      for (const analysis of analyses) {
+        res += analysis?.name.split('/')[1]
+        if (i < end) {
+          res += ', '
+        }
+        i++
+      }
+      return res
+    }
+
+    return h(Modal, {
+      onDismiss,
+      width: 530,
+      title: outdatedAnalyses?.length > 1 ? 'R Markdown Files In Use' : `R Markdown File Is In Use`,
```

Comment:
Can we use Lodash like so 

```suggestion
      title: _.isEmpty(outdatedAnalyses) ? 'R Markdown Files In Use' : `R Markdown File Is In Use`,
```

for reasons explained in the comment above?

Similarly for other occurences of `outdatedAnalyses?.length > 1`.

Reply:
Have we checked with UX as far as the casing of the text; i.e., are we sure it shouldn't be `R markdown files in use`?

Reply:
so this check is actually checking if outdatedAnalyses has 1 or more than 1 entry in it so `_.isEmpty` would not work. maybe let's sync up when we pair on a better implementation.

And yeah, UX has approved this modal. We have a similar modal for notebooks whose title is `Notebook Is In Use`